Formula/f/fail2ban.rb

```diff
@@ -76,8 +76,8 @@ def install
 
   def inreplace_etc_var(targets, audit_result: true)
     inreplace targets do |s|
-      s.gsub! %r{/etc}, etc, audit_result
-      s.gsub! %r{/var}, var, audit_result
+      s.gsub! %r{/etc}, etc, audit_result: audit_result
+      s.gsub! %r{/var}, var, audit_result: audit_result
```

Comment:
We can use a shorthand if you want
```suggestion
      s.gsub!(%r{/etc}, etc, audit_result:)
      s.gsub!(%r{/var}, var, audit_result:)
```

Reply:
Ah, I tried this initially without the brackets and it didn't work. Didn't realise that the brackets are needed. Thanks!